app/components/elements/forms/checkbox_component.rb

```diff
@@ -4,6 +4,9 @@ module Elements
   module Forms
     # Component for form checkbox field
     class CheckboxComponent < FieldComponent
+      def checked?
+        form.object.public_send(field_name)
```

Comment:
Is this needed? Isn't this done by `form.checkbox`?

Reply:
Good call, yes!